src/main/java/org/commonjava/indy/service/tracking/config/IndyTrackingConfiguration.java

```diff
@@ -31,6 +31,9 @@ public interface IndyTrackingConfiguration
     @WithName( "baseDir" )
     File baseDir();
 
+    @WithName( "main-indy" )
```

Comment:
It would be better to use the one for example `content-service-api`.

Reply:
Or content-service.url?

Reply:
yeah. +1

Reply:
updated.